ui/src/pages/providers/ProviderCreate.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { FC } from "react";
+import {
+  ActionButton,
+  Button,
+  Col,
+  Row,
+  useNotify,
+} from "@canonical/react-components";
+import { useFormik } from "formik";
+import * as Yup from "yup";
+import { queryKeys } from "util/queryKeys";
+import { useQueryClient } from "@tanstack/react-query";
+import { useNavigate } from "react-router-dom";
+import ProviderForm, { ProviderFormTypes } from "pages/providers/ProviderForm";
+import { createProvider } from "api/provider";
+import SidePanel from "components/SidePanel";
+
+const ProviderCreate: FC = () => {
+  const navigate = useNavigate();
+  const notify = useNotify();
+  const queryClient = useQueryClient();
+
+  const ProviderCreateSchema = Yup.object().shape({
+    id: Yup.string().required("This field is required"),
+  });
+
+  const formik = useFormik<ProviderFormTypes>({
+    initialValues: {
+      client_secret: "secret-9",
+      id: "okta_347646e49b484037b83690b020f9f629",
+      client_id: "347646e4-9b48-4037-b836-90b020f9f629",
+      provider: "okta",
+      mapper: "file:///etc/config/kratos/okta_schema.jsonnet",
+      scope: "email",
```

Comment:
issue: hardcoding values, **especially** `client-secret` should not appear in any frontend code to avoid leaks

Reply:
Right, those values should all be empty string. Those have values atm to ease testing.

Reply:
Totally agree, thank you!

Reply:
Removed the defaults here.